examples/machine_vision_object_pickup/main.py

```diff
@@ -1,89 +1,91 @@
-# Automata Technologies - Andrea Antonello, 2019
+# Automata Technologies - Dr. Andrea Antonello, 2019
 import socket
 from automata import Eva
 from evaUtilities import *
 
-# Connection to robot
-host = <user_input>
-token = <user_input>
-eva = Eva(host, token)
-eva.lock_status()
 
-# Connection to camera
-sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-server = <user_input>
-port = <user_input>
-sock.connect((server, port))
+if __name__ == "__main__":
+    # Connection to robot
+    host = <user_input>
```

Comment:
If you don't want to provide default, it's fine but you should at least extract all of them in another file or in a central place, hunting for them in different parts of the code is a pain (maybe add a single comment for each about what data you expect)

Reply:
I created a new commit with a config folder and a .yaml file containing all the setup parameters

Reply:
Seems slightly overkill but that works!